Libraries/PeriphDrivers/Source/I2C/i2c_me10.c

```diff
@@ -113,7 +103,7 @@ int MXC_I2C_Shutdown(mxc_i2c_regs_t *i2c)
 /* ************************************************************************** */
 int MXC_I2C_SetFrequency(mxc_i2c_regs_t *i2c, unsigned int hz)
 {
-    if (hz > MXC_I2C_FAST_SPEED) {
+    if (hz > MXC_I2C_REVB_FASTPLUS_SPEED) {
```

Comment:
This works but not sure if it's necessary to use _REVB_ definitions in the part-specific library file.

Reply:
Agreed, should use `MXC_I2C_FASTPLUS_SPEED` here instead